cmd/migrate.go

```diff
@@ -22,17 +46,35 @@ func Migrate() {
 		SSLMode:  config.DBSSLMode,
 	})
 	if err != nil {
-		panic(err)
+		return err
 	}
 
-	fmt.Println("Migrating DB...")
+	log.Info("Migrating DB...")
 	if err := pgClient.AutoMigrate(
 		&postgres.Template{},
 		&postgres.Field{},
 		&postgres.Tag{},
 	); err != nil {
-		panic(err)
+		return err
 	}
 
-	fmt.Println("Migration done.")
+	return nil
+}
+
+func migrateElasticsearch() (err error) {
+	log.Info("Initiating ES client...")
+	esClient := initElasticsearch(config)
+	tr := esStore.NewTypeRepository(esClient)
+	for _, supportedType := range record.AllSupportedTypes {
+		log.Infof("Migrating %q type\n", supportedType.Name)
+		ctx, cancel := context.WithTimeout(context.Background(), esMigrationTimeout)
+		defer cancel()
+		err = tr.CreateOrReplace(ctx, supportedType)
+		if err != nil {
+			err = fmt.Errorf("error creating/replacing type: %q, err: %w", supportedType.Name, err)
```

Comment:
we can change this to 
```suggestion
			err = errors.Wrapf(err, "error creating/replacing type: %q", supportedType.Name)
```